src/pages/ProductList/ProductList.js

```diff
@@ -3,7 +3,185 @@ import './ProductList.scss';
 
 class ProductList extends Component {
   render() {
-    return <div></div>;
+    return (
+      <div>
+        <html>
+          <body>
+            <section>
+              <div className="menuPath">
+                <span>홈</span>
+                <span className="menuPathSlash">/</span>
+                <span>Men</span>
+                <span className="menuPathSlash">/</span>
+                <span>의류</span>
+              </div>
+              <header className="categoryHeader">
+                <img
+                  className="categoryIntroPic"
+                  alt="categoryIntroImage"
+                  src="https://imageapac1.lacoste.com/dw/image/v2/BBCL_PRD/on/demandware.static/-/Library-Sites-LacosteContent/default/dw7a5def12/2021/FW21/PLP/plp-men-clothing-desk-01.jpg?imwidth=915&impolicy=custom"
+                />
+                <div className="categoryIntro">
+                  <div className="categoryTitle">남성 폴로</div>
+                  <div className="categoryDescription">
+                    <div>
+                      라코스테의 아이코닉한 클래식핏, 매끈하게 떨어지는 슬림핏
+                    </div>
+                    <div>
+                      혹은 우아한 셔츠같은 파리 폴로까지 어떤 핏을 선택하든
+                    </div>
+                    <div>라코스테 폴로는 후회하지 않을 선택일거예요.</div>
+                  </div>
+                  <div className="subCategoryBox">
+                    <div className="subCategoryBoxFirst">
+                      <li>폴로</li>
+                      <li>파리폴로</li>
+                    </div>
+                    <div className="subCategoryBoxSecond">
+                      <li>클래식핏</li>
+                      <li>레귤러핏</li>
+                    </div>
+                  </div>
+                  <div className="filterFunction">
+                    <div className="filteringResult">58개의 결과</div>
+                    <div className="filterAndAlign">
+                      <i class="fas fa-sliders-h"></i> 필터 및 정렬
+                    </div>
```

Comment:
```suggestion
                  <div className="subCategories">
                    <div className="subCategory">
                      <li>폴로</li>
                      <li>파리폴로</li>
                    </div>
                    <div className="subCategory">
                      <li>클래식핏</li>
                      <li>레귤러핏</li>
                    </div>
                  </div>
                  <div className="filter">
                    <div className="filteResult">58개의 결과</div>
                    <div className="filterAndAlign">
                      <i class="fas fa-sliders-h"></i> 필터 및 정렬
                    </div>
```
- 목록 같은 경우 list, 감싸는 요소는 wrapper, container 같은 네이밍을 쓰긴 하지만 위와 같이 단수 - 복수 관계로 간단히 표현해주어도 나쁘지 않을 것 같습니다.
- filterFunction이라는 이름이 가리키는 내용은 실제 함수가 아니기 때문에 혼동을 줄 여지가 있습니다. 간단히 filter로 바꾸거나 productFilter 같은 이름이 좋겠습니다.